framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java

```diff
@@ -241,6 +246,54 @@ private void executeQuery(String query) throws IOException, SQLException {
     }
   }
 
+  private void executeLimitZeroQuery(String query) throws IOException, SQLException {
+	if (getTestStatus() == TestStatus.CANCELED) {
```

Comment:
Should we run the LIMIT 0 query only if the test status is PASS? This will avoid dealing with multiple errors.